src/pages/UnlockedWallet/OverviewPage/GreetingMessages.tsx

```diff
@@ -48,7 +48,8 @@ const GreetingMessages = ({ className }: GreetingMessagesProps) => {
 
   const priceComponent = (
     <span key="price">
-      {price ? '📈 ' + t('ALPH price: {{ price }}$', { price: formatFiatAmountForDisplay(price) }) : ''}
+      {price ? '📈 ' + t('ALPH price: {{ price }}', { price: formatFiatAmountForDisplay(price) }) : ''}
+      {currencies[fiatCurrency].symbol}
```

Comment:
hm, so, if there is no `price`, then just the symbol is being displayed?

Reply:
I think it'd be better to do

```ts
{price ? '📈 ' + t('ALPH price: {{ price }}', { price: formatFiatAmountForDisplay(price) }) + currencies[fiatCurrency].symbol : ''}
```

or a bit cleaner (imo)

```ts
{price ? `📈  ${t('ALPH price: {{ price }}', { price: formatFiatAmountForDisplay(price) })} ${currencies[fiatCurrency].symbol}` : ''}
```

Reply:
💜

Reply:
(I didn't see your previous comment before commiting)